sdk/arc-scvmm/Azure.ResourceManager.ScVmm/samples/Generated/Samples/Sample_ScVmmInventoryItemCollection.cs

```diff
@@ -91,7 +91,7 @@ public async Task Get_GetInventoryItem()
         public async Task GetAll_InventoryItemsListByVMMServer()
         {
             // Generated from example definition: specification/scvmm/resource-manager/Microsoft.ScVmm/stable/2023-10-07/examples/ListInventoryItemsByVMMServer.json
-            // this example is just showing the usage of "InventoryItems_ListByVmmServer" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "InventoryItems_ListByVMMServer" operation, for the dependent resources, they will have to be created separately.
```

Comment:
seems a regression, should be `Vmm`: https://github.com/Azure/azure-rest-api-specs/blob/c852a76b96e5c71972cdc62a303665fec7c867b6/specification/scvmm/ScVmm.Management/InventoryItem.tsp#L40